### PR TITLE
Wqp 1789 - Moved utilitiy classes to style sheets, cleaned up the index.html markup and made the form more mobile friendly

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/js/views/DownloadFormView.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default {
   props: ['form', 'downloadProgressDialog', 'downloadProgressDialogBasic', "providers"],
   data () {
     return {
-      selectedForm: document.querySelector('#paramsBasic')
+      selectedForm: document.querySelector('#params-basic')
```
Comment (abriggs-usgs) [inline assets/js/views/DownloadFormView.vue:45]: Nice. I appreciate this change. :)

File: assets/scss/custom/layout.scss [hunk 2]
```diff
@@ -1,12 +1,8 @@
-// custom/layout.scss
-// custom layout CSS properties
-@import "uswds";
 
 // Center page, max width
 .page-center {
     display: block;
     box-sizing: border-box;
-    //background-color: white;
```
Comment (abriggs-usgs) [inline assets/scss/custom/layout.scss:9]: Dang it, I missed that one.

File: assets/scss/partials/_portal-form.scss [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    outline: none !important;
+  }
+
+  .multiselect:focus-within {
+    @include u-outline-color('blue-40v');
+    @include u-outline('05');
+  }
+
+  .multiselect__tag, .multiselect__option--highlight, .multiselect__option--selected.multiselect__option--highlight {
+    @include u-bg('blue-60v', !important);
```
Comment (abriggs-usgs) [inline assets/scss/partials/_portal-form.scss:20]: I wonder if the !important can be removed now?

Comment (mbucknell) [reply assets/scss/partials/_portal-form.scss:20]: I can give it a try

Comment (mbucknell) [reply assets/scss/partials/_portal-form.scss:20]: It will likely be in the next PR, though

**Review by abriggs-usgs — APPROVED** [review]
Looks good.

**Pull request merged**